src/main/java/org/springframework/samples/petclinic/model/Card.java

```diff
@@ -0,0 +1,45 @@
+package org.springframework.samples.petclinic.model;
+
+import java.util.Set;
+
+import org.springframework.samples.petclinic.model.enums.Icon;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cards")
+public class Card {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.UUID)
+  private String id;
+
+  @Size(min = 8, max = 8)
+  @Enumerated(EnumType.STRING)
+  @Column(name = "icon")
+  private Set<Icon> icons;
```

Comment:
```suggestion
  private List<Icon> icons;
```

Si no guardamos la posición, entonces tenemos que guardar esto como un conjunto ordenado. No veo donde está la rotación tampoco?

Reply:
Si la rotación la tengo que añadir también aunque ns si es muy necesaria

Reply:
Es cuestión de meter ``private Integer rotation = new Random().nextInt(361)`` en la entidad que sea para obtener un número aleatorio en cuanto se instancie una nueva clase entre 0 y 360. Lo aplicamos al cliente también y así cada carta es única.

Es el menor de nuestros problemas ahora mismo, facilisimo de implementar y mejora mucho la experiencia.